packages/client/hooks/useIsIntegrated.ts

```diff
@@ -0,0 +1,97 @@
+import graphql from 'babel-plugin-relay/macro'
+
+import {useIsIntegrated_integrations$key} from '../__generated__/useIsIntegrated_integrations.graphql'
+import {useFragment} from 'react-relay'
+
+type IntegrationLookup = {
+  hasGitHub: boolean
+  hasAtlassian: boolean
+  hasGitLab: boolean
+  hasJiraServer: boolean
+  hasAzureDevOps: boolean
+}
+
+graphql`
+  fragment useIsIntegratedJiraServerIntegration on JiraServerIntegration {
+    auth {
+      isActive
+    }
+  }
+`
+graphql`
+  fragment useIsIntegratedAtlassianIntegration on AtlassianIntegration {
+    isActive
+  }
+`
+graphql`
+  fragment useIsIntegratedGitHubIntegration on GitHubIntegration {
+    isActive
+  }
+`
+graphql`
+  fragment useIsIntegratedGitLabIntegration on GitLabIntegration {
+    auth {
+      isActive
+    }
+  }
+`
+graphql`
+  fragment useIsIntegratedAzureDevOpsIntegration on AzureDevOpsIntegration {
+    auth {
+      isActive
+    }
+  }
+`
+
+export const makePlaceholder = (integrationLookup: IntegrationLookup) => {
+  const {hasGitHub, hasAtlassian, hasGitLab, hasAzureDevOps} = integrationLookup
+  const names = [] as string[]
+  if (hasGitHub) names.push('GitHub')
+  if (hasAtlassian) names.push('Jira')
+  if (hasGitLab) names.push('GitLab')
+  if (hasAzureDevOps) names.push('Azure DevOps')
+  return `Search ${names.join(' & ')}`
```

Comment:
+1 consider using `Intl.ListFormat` for such things, [an example here](https://github.com/ParabolInc/parabol/blob/34080336e5be425eae060bfa7a916d333fff47bb/packages/client/components/ReflectionCard/EmojiUsersReaction.tsx#L26)

Reply:
This is actually just copy pasted as part of the refactor, so I'd prefer to leave this as-is for now